preparation.py

```diff
@@ -31,6 +33,9 @@ def overlay_masks(data_dir, dataset, target_dir, category_ids, is_small=False):
                 annotation_ids = coco.getAnnIds(imgIds=image_id, catIds=[category_id, ])
                 annotations = coco.loadAnns(annotation_ids)
                 mask = overlay_masks_from_annotations(annotations, image_size)
```

Comment:
@apyskir it seems that this mask is generated regardles of erosion, which means that if erode>0 the calculations done here are redundant.

Reply:
@jakubczakon No, they are not redundant. We need not eroded mask in line 38 to pass it to the function add_dropped_objects().